mlflow/models/evaluation/base.py

```diff
@@ -152,27 +147,27 @@ def make_metric(
         .. code-block:: python
 
             def eval_fn(
-                eval_df: Union[pandas.Dataframe, pyspark.sql.DataFrame],
+                predictions: pandas.Series,
```

Comment:
Is this always a series? It could be a DataFrame if the model returns multiple columns as output, right?

Reply:
Currently we assume that y_pred is always a series. 
https://github.com/mlflow/mlflow/blob/mlflow-evaluation/mlflow/models/evaluation/default_evaluator.py#L439-L463
So not sure if we support df as predictions right now. 
Also are there examples currently that would have outputs to be more than 1 column?